scripts/post-install.sh

```diff
@@ -60,31 +58,41 @@ fi
 # Distribution-specific logic
 if [[ -f /etc/redhat-release ]]; then
     # RHEL-variant logic
-    which systemctl &>/dev/null
-    if [[ $? -eq 0 ]]; then
-	    install_systemd
+    if pidof systemd &>/dev/null; then
```

Comment:
Tried that. Indeed it's better and i didn't knew before that systemd can also run but not be the init system. will change the code accordingly ;)